Source: Upstream
Upstream: Yes
Reason: Fix build failure with recent phonon (?)

From 5942553287f33b31e83369f0b86f327b17d2db32 Mon Sep 17 00:00:00 2001
From: woebbe <woebbe@283d02a7-25f6-0310-bc7c-ecb5cbfe19da>
Date: Sun, 14 Nov 2010 17:52:50 +0000
Subject: [PATCH] compile (phononnamespace.h isn't local)

git-svn-id: svn+ssh://svn.kde.org/home/kde/branches/KDE/4.5/kdebase/runtime@1197039 283d02a7-25f6-0310-bc7c-ecb5cbfe19da
---
 phonon/kded-module/phononserver.cpp   |    2 +-
 phonon/platform_kde/devicelisting.cpp |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/phonon/kded-module/phononserver.cpp b/phonon/kded-module/phononserver.cpp
index 8d6fb40..a8d2459 100644
--- a/phonon/kded-module/phononserver.cpp
+++ b/phonon/kded-module/phononserver.cpp
@@ -18,7 +18,6 @@
 
 */
 
-#include "phononnamespace.h"
 #include "phononserver.h"
 #include "audiodevice.h"
 
@@ -45,6 +44,7 @@
 #include <Solid/GenericInterface>
 #include <Solid/Device>
 #include <Solid/DeviceNotifier>
+#include <phonon/phononnamespace.h>
 
 #include <../config-alsa.h>
 #ifdef HAVE_LIBASOUND2
diff --git a/phonon/platform_kde/devicelisting.cpp b/phonon/platform_kde/devicelisting.cpp
index e2aa5c3..1130c59 100644
--- a/phonon/platform_kde/devicelisting.cpp
+++ b/phonon/platform_kde/devicelisting.cpp
@@ -19,7 +19,6 @@
 */
 
 #include "devicelisting.h"
-#include "phononnamespace.h"
 
 #include <QtCore/QFile>
 #include <QtDBus/QDBusReply>
@@ -29,6 +28,7 @@
 #include <kdebug.h>
 #include <klocale.h>
 #include <ksharedconfig.h>
+#include <phonon/phononnamespace.h>
 
 #include <../config-alsa.h>
 #ifdef HAVE_ALSA_ASOUNDLIB_H
-- 
1.7.3.2

